hyperion/grid/voronoi_grid.py

```diff
@@ -450,10 +456,17 @@ def write(self, group, quantities='all', copy=True, absolute_paths=False, compre
         voronoi_table['volume'][np.isnan(voronoi_table['volume'])] = -1.
         voronoi_table['volume'][np.isinf(voronoi_table['volume'])] = -1.
 
-        if voronoi_table['neighbours'].dtype != np.int32:
-            raise TypeError("neighbours should be int32")
+        # Remove the dense neighbours from the table.
+        voronoi_table = voronoi_table['coordinates', 'volume', 'bb_min', 'bb_max']
```

Comment:
I think there is some visualisation functionality which currently uses the dense representation. I don't think it would affect much else. Would you like me to remove it?